WcaOnRails/app/models/event.rb

```diff
@@ -22,6 +22,10 @@ def name_in(locale)
     I18n.t(id, scope: :events, locale: locale)
   end
 
+  def cellName
+    name
+  end
```

Comment:
IIRC the plan was to remove the `cellName` column in the future, so I'd suggest changing all `#cellName` calls to `#name` and getting rid of this method.